package.json

```diff
@@ -25,7 +25,7 @@
     "predocs-test": "npm run docs",
     "docs": "typedoc --readme none && touch docs/.nojekyll",
     "docs-deploy": "gh-pages --dotfiles --dist docs",
-    "docs:test": "linkinator docs --silent --retry && linkinator doc/*.md --silent --retry",
+    "docs:test": "linkinator docs --silent --retry && linkinator doc/*.md --skip '^(?!http://localhost)' --silent --retry",
```

Comment:
@dyladan hmm looks like this is not working correctly. For me this filters out https://opentelemetry.io/nonexistent and still includes localhost. :thinking: 

Tried coming up with a regex that would filter just `localhost` ones but it does not seem to work how I think it would. :confused:

Reply:
Apparently localhost causes problems because linkinator starts a local server to serve the local files. I added the specific ports and it works as expected.